CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ## [[NEXT]](https://github.com/iExecBlockchainComputing/iexec-commons-poco/releases/tag/vNEXT) 2025
 
+### Quality
+
+- Update source and target compatibility to Java 17. (#114)
+- Migrated to Jakarta EE APIs. (#114)
+
+### Dependency Upgrades
+
+- Upgrade Spring Boot to `3.0.13`. (#114)
+- Upgrade XML Binding to `jakarta.xml.binf:jakarta.xml.bind-api`. (#114)
```

Comment:
I think this can be omitted, being part of Spring Boot 3.0.13 migration. It is a bit strange to only mention this dependency and not the other ones.